src/vnmr/part11.c

```diff
@@ -5607,7 +5615,10 @@ int save_optFilesInParam(char* dest, char *param, char *type)
            }
            while(fileExist(path1) && !(fp = fopen(path1, "r"))) {
             sleep(1);
+            fp = NULL;
```

Comment:
fp=NULL will always be true here too.  the whole loop is weird though, it's not guaranteed that because a file exists you will eventually be able to open it for reading, maybe it should timeout after some reasonable period.

Reply:
Yes, the fp will be NULL. I added them just to keep cppcheck from complaining. Also, I agree, the loop is a little weird. However, I did not want to change the logic because this part11 stuff is difficult to test and it has been working.